### PR TITLE
Set pool programs

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

File: packages/solidity/contracts/StakingRewardsStore.sol [hunk 1]
```diff
@@ -46,7 +52,7 @@ contract StakingRewardsStore is IStakingRewardsStore, AccessControl, Utils, Time
      * @param poolToken the pool token representing the rewards pool
      * @param startTime the starting time of the program
      * @param endTime the ending time of the program
-     * @param rewardRate the program's weekly rewards
+     * @param rewardRate the program's rewards rate
```
Comment (yudilevi) [inline packages/solidity/contracts/StakingRewardsStore.sol:55]: unclear how granular it is - rate per second?

File: packages/solidity/contracts/StakingRewardsStore.sol [hunk 2]
```diff
@@ -103,24 +143,24 @@ contract StakingRewardsStore is IStakingRewardsStore, AccessControl, Utils, Time
...
     /**
-     * @dev returns whether the specified pool  is participating in the rewards program
+     * @dev returns whether the specified pool is participating in the rewards program
      *
      * @param program the program data
      *
-     * @return whether the specified pool  is participating in the rewards program
+     * @return whether the specified pool is participating in the rewards program
      */
     function isPoolParticipating(PoolProgram memory program) private view returns (bool) {
```
Comment (yudilevi) [inline packages/solidity/contracts/StakingRewardsStore.sol:152]: this seems a bit excessive - why not just use the normal `isPoolParticipating` function that receives the pool token?

File: packages/app/programs.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "poolToken": "0x6b181C478b315bE3f9E99c57CE926436c32e17a7",
+        "networkToken": "0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c",
```
[the remark this answers is not in the document]
Comment (lbeder) [reply packages/app/programs.json:4]: Not at all, but it is useful sometimes when analyzing the data manually

File: packages/solidity/contracts/StakingRewardsStore.sol (hunk 1 above)
Comment (lbeder) [reply packages/solidity/contracts/StakingRewardsStore.sol:55]: Rate per-second, yeah. Do you want me to add this explicitly?

File: packages/app/programs.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (yudilevi) [reply packages/app/programs.json:4]: 👍

File: packages/solidity/contracts/StakingRewardsStore.sol (hunk 2 above)
Comment (lbeder) [reply packages/solidity/contracts/StakingRewardsStore.sol:152]: I wanted to save some gas between other calls, but I can remove it

File: packages/solidity/contracts/StakingRewardsStore.sol (hunk 1 above)
Comment (yudilevi) [reply packages/solidity/contracts/StakingRewardsStore.sol:55]: in the comments? yeah

Comment (lbeder) [reply packages/solidity/contracts/StakingRewardsStore.sol:55]: 👍

File: packages/solidity/contracts/StakingRewardsStore.sol (hunk 2 above)
Comment (yudilevi) [reply packages/solidity/contracts/StakingRewardsStore.sol:152]: the whole thing is very inexpensive so I'd remove it, less code to worry about

Comment (lbeder) [reply packages/solidity/contracts/StakingRewardsStore.sol:152]: 👍

**Review by yudilevi — APPROVED** [review]

**Pull request merged**